_data/navigation.js

```diff
@@ -1,230 +1,80 @@
 module.exports = {
   navigation: {
-    nodeOperator: [
+    gettingStarted: [
       {
-        section: 'NODE OPERATORS',
+        section: 'Getting Started',
         contents: [
           {
-            url: '/docs/node-versions/',
-            title: 'Node Versions' },
-          {
-            url: '/docs/running-a-chainlink-node/',
-            title: 'Running a Chainlink Node',
+            title: 'Conceptual Overview',
+            url: '/docs/conceptual-overview/',
           },
           {
-            url: '/docs/fulfilling-requests/',
-            title: 'Fulfilling Requests' },
-          {
-            url: '/docs/run-an-ethereum-client/',
-            title: 'Run an Ethereum Client',
+            title: 'Deploy Your First Contract',
+            url: '/docs/deploy-your-first-contract/',
           },
           {
-            url: '/docs/performing-system-maintenance/',
-            title: 'Performing System Maintenance',
+            title: 'Consuming Data Feeds',
+            url: '/docs/consuming-data-feeds/',
           },
           {
-            url: '/docs/connecting-to-a-remote-database/',
-            title: 'Connecting to a Remote Database',
+            title: 'Get Random Numbers',
+            url: '/docs/intermediates-tutorial/',
```

Comment:
It looks like there's no Any API 'getting started'. I think that's important as this is the #1 product developers actually want to use for things like hackathons. How can we help thoes users?

Reply:
Restored from the main docs to Getting Started for now.